### ISSUE TITLE
pudae

### ISSUE THREAD

**Issue by zzzl-h** [pudae/tensorflow-densenet#13]
hi, 
when I restore your pre_trained model, there raise some mistake

NotFoundError (see above for trackback): Key densenet121/dense_block1/conv_block2/*1/Conv/biases not found in the checkpoint

here is my code

x=tf.placeholder("float",[None,224,224,3])
y=tf.placeholder("float",[None,num_classes])

logits, collection= densenet.densenet121(x,num_classes=num_classes)
predictions = collection['predictions']
loss=-tf.reduce_sum(y*tf.log(predictions))
optimizer=tf.train.GradientDescentOptimizer(learning_rate=learning_rate).minimize(loss)
accuracy=tf.reduce_mean(tf.cast(tf.equal(tf.argmax(predictions, 1), tf.argmax(y, 1)),"float"))

saver=tf.train.Saver()
with tf.Session() as sess:
    saver.restore(sess,"tf-densenet121.ckpt")

**Comment by pudae** [comment]
Please, refer to the issue [#10].
Thanks~
